### PR TITLE
Enhancement of training classification section

### PR DESCRIPTION
**Pull request by thibaultdvx**
Typos, punctuation, layout and clarifications

### CODE REVIEW

**Review by NicolasGensollen — COMMENTED** [review]
Thanks @thibaultdvx !
Two small comments/questions, but LGTM otherwise !

File: src/training_classification.py [hunk 1]
```diff
@@ -87,45 +85,45 @@
 # deeplearning_prepare_data
 # ├── image_based
 # │   └── t1_linear
-# │       └── sub-<participant_label>_ses-<session_label>_T1w_space-MNI152NLin2009cSym_desc-Crop_res-1x1x1_T1w.pt
+# │       └── sub-<participant_label>_ses-<session_label>_space-MNI152NLin2009cSym_desc-Crop_res-1x1x1_T1w.pt
```
[the remark this answers is not in the document]
Comment (thibaultdvx) [reply src/training_classification.py:88]: Hi @NicolasGensollen! When I run the handbook, I get the files without `T1w` before `_space'. I don't have the prefix `T1w` for my files in the CAPS directory either. It seems that there is no problem with this specific suffix, at least for the `t1_linear` pipeline. What do you think?

[the remark this answers is not in the document]
Comment (NicolasGensollen) [reply src/training_classification.py:88]: I'll double check that. Maybe I forgot about it... 🤔

[the remark this answers is not in the document]
Comment (NicolasGensollen) [reply src/training_classification.py:88]: You're absolutely right ! See https://github.com/aramis-lab/clinica/pull/940

I totally forgot that we fixed that already... 😓

File: src/training_classification.py [hunk 2]
```diff
@@ -144,14 +142,14 @@
 # %% [markdown]
 # ## Before starting
 # If you failed to obtain the preprocessing using the `t1-linear` pipeline,
-# please uncomment the next cell. You can extract tensors from this CAPS but
+# please uncomment the next cell. You can extract tensors from this CAPS, but
 # for the training part you will need a bigger dataset.
 # %%
-# !curl -k https://aramislab.paris.inria.fr/clinicadl/files/handbook_2023/data_oasis/CAPS_example_prepared.tar.gz -o oasisCaps.tar.gz
+# !curl -k https://aramislab.paris.inria.fr/clinicadl/files/handbook_2023/data_oasis/CAPS_example.tar.gz -o oasisCaps.tar.gz
```
[the remark this answers is not in the document]
Comment (thibaultdvx) [reply src/training_classification.py:148]: @NicolasGensollen yes I'm pretty sure. `CAPS_example.tar.gz` is the CAPS folder obtained after `t1_linear` pipeline, whereas `CAPS_extracted.tar.gz`is the CAPS folder obtained after `prepare-data`. The latter can be downloaded line 167, if the user failed to run `prepare-data`.

[the remark this answers is not in the document]
Comment (thibaultdvx) [reply src/training_classification.py:148]: @NicolasGensollen yes I'm pretty sure. `CAPS_example.tar.gz` is the CAPS folder obtained after `t1_linear` pipeline, whereas `CAPS_extracted.tar.gz`is the CAPS folder obtained after `prepare-data`. The latter can be downloaded line 167, if the user failed to run `prepare-data`.

[the remark this answers is not in the document]
Comment (thibaultdvx) [reply src/training_classification.py:148]: But then I should modify line 167, because `CAPS_extracted.tar.gz` doesn't exist! I missed that, thanks!

**Review by NicolasGensollen — APPROVED** [review]
LGTM, thanks !

**Pull request merged**